packages/medusa/src/api/routes/admin/price-lists/modules-queries/get-price-list.ts

```diff
@@ -12,9 +12,7 @@ export async function getPriceListPricingModule(
   }
 ): Promise<PriceList> {
   const [priceLists, count] = await listAndCountPriceListPricingModule({
-    filters: {
-      id: [id],
-    },
+    filters: { id: [id] },
```

Comment:
Doesn't this work if it is `filters: { id }`?

Reply:
It will, this file will be deleted soon though 😄 its legacy stuff. Just waiting on one more endpoint before I nuke this.